nncf/layer_utils.py

```diff
@@ -14,6 +14,8 @@ def __getattr__(self, name):
 
 
 class _NNCFModuleMixin:
+    target_compression_weight_dim = 0
```

Comment:
`target_weight_dim_for_compression`

Reply:
Also, I expect you to utilize the same mechanism for correct `ConvTranspose` per-channel quantization, perhaps in a separate PR.

Reply:
Renamed.
Yes, i will do it in another PR.